runtime_builders/publish_builders.py

```diff
@@ -32,6 +34,8 @@ def main():
                         required=True)
     args = parser.parse_args()
 
+    manifest = {}
```

Comment:
Now that the manifest is a shared file for all runtimes, I think you need to start by loading what exists or else you will blow away all the other runtime settings when you update.